client/ayon_maya/plugins/create/create_render.py

```diff
@@ -12,7 +12,7 @@
 )
 
 
-class CreateRenderlayer(plugin.RenderlayerCreator):
+class CreateRender(plugin.RenderlayerCreator):
```

Comment:
Do we need to implement the backward compatibility for `CreateRenderLayer` or?

Reply:
As far as I know, we don't. We either change the class name or set `settings_name` attribute. But the class name isn't stored or imprinted anywhere as far as I know, we always refer to the creator identifiers instead so the class name should be safe to rename.